repos/fdbt-site/src/pages/api/ticketConfirmation.ts

```diff
@@ -1,9 +1,24 @@
 import { NextApiResponse } from 'next';
 import { NextApiRequestWithSession } from '../../interfaces/index';
 import { redirectTo, redirectToError } from '../../utils/apiUtils';
+import { FARE_TYPE_ATTRIBUTE } from 'src/constants/attributes';
+import { getSessionAttribute } from 'src/utils/sessions';
+import { isFareType } from 'src/interfaces/typeGuards';
 
-export default (_req: NextApiRequestWithSession, res: NextApiResponse): void => {
+export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
     try {
+        const fareTypeAttribute = getSessionAttribute(req, FARE_TYPE_ATTRIBUTE);
```

Comment:
if you use getFareTypeFromFromAttributes function, it does all the error handling for you.